bluesea_node:
  ros__parameters:
    type: "uart"
    platform: "LDS-50C-2"
    port: "/dev/ttyUSB0"
    frame_id: "map"
    rate_list: "768000, 1000000"
    baud_rate: 768000
    raw_bytes: 2
    max_dist: 25.0
    output_360: true
    output_scan: true
    output_cloud: false
    with_angle_filter: false
    min_angle: -3.14159
    max_angle: 3.14159
    hard_resample: false
    soft_resample: true
    resample_res: 0.300
    unit_is_mm: true
    with_confidence: true
    with_checksum: true
    rpm: 900
    with_smooth: true
    with_deshadow: true
    inverted: false
    reversed: false
    topic: "scan"
    lidar_ip: "192.168.158.91"
    lidar_port: 6543
    local_port: 6666
    group_ip: "224.1.2.3"
    dev_id: 0
    alarm_msg: false